algo/src/main/java/com/hartwig/serve/sources/ckb/blacklist/CkbBlacklistMolecularProfileFile.java

```diff
@@ -8,35 +8,35 @@
 import org.apache.commons.compress.utils.Lists;
 import org.jetbrains.annotations.NotNull;
 
-public final class CkbFilterFile {
+public final class CkbBlacklistMolecularProfileFile {
 
     private static final String FIELD_DELIMITER = "\t";
 
-    private CkbFilterFile() {
+    private CkbBlacklistMolecularProfileFile() {
     }
 
     @NotNull
-    public static List<CkbFilterEntry> read(@NotNull String ckbFilterTsv) throws IOException {
+    public static List<CkbBlacklistMolecularProfileEntry> read(@NotNull String ckbFilterTsv) throws IOException {
```

Comment:
You missed one rename here! 

Would be good to do a global search on `ckbFilter` and replace remaining cases

Reply:
Good catch, have now everything renamed (If my search missed nothing)